<use f='include/llvm-6.0/llvm/Support/Error.h' l='1124' u='c' c='_ZNK4llvm11ExitOnErrorclENS_5ErrorE'/>
<use f='include/llvm-6.0/llvm/Support/Error.h' l='1129' u='c' c='_ZNK4llvm11ExitOnErrorclEONS_8ExpectedIT_EE'/>
<use f='include/llvm-6.0/llvm/Support/Error.h' l='1136' u='c' c='_ZNK4llvm11ExitOnErrorclEONS_8ExpectedIRT_EE'/>
<def f='include/llvm-6.0/llvm/Support/Error.h' l='1141' ll='1147' type='void llvm::ExitOnError::checkError(llvm::Error Err) const'/>
